src/test/java/uk/gov/ons/census/fwmtadapter/messaging/RefusalReceiverIT.java

```diff
@@ -139,4 +140,44 @@ public void testRefusalMessageFromFieldChannelDoesNotEmitMessageToField()
     // Then
     assertThat(rabbitQueueHelper.getMessage(outboundQueue)).isNull();
   }
+
+  @Test
+  public void testRefusalFromNonFieldChannelForEstabAddressLevelCaseDoesNotEmitToField()
+      throws InterruptedException, JsonProcessingException {
+    // Given
+    BlockingQueue<String> outboundQueue = rabbitQueueHelper.listen(ADAPTER_OUTBOUND_QUEUE);
+
+    CollectionCase collectionCase = new CollectionCase();
+    collectionCase.setId(TEST_CASE_ID);
+    Refusal refusal = new Refusal();
+    refusal.setCollectionCase(collectionCase);
+    Payload payload = new Payload();
+    payload.setRefusal(refusal);
+    ResponseManagementEvent responseManagementEvent = new ResponseManagementEvent();
+    responseManagementEvent.setPayload(payload);
+    Event event = new Event();
+    event.setType(EventType.REFUSAL_RECEIVED);
+    event.setChannel("CC");
+    responseManagementEvent.setEvent(event);
+
+    String url = "/cases/" + TEST_CASE_ID;
+    CaseContainerDto caseContainerDto = new CaseContainerDto();
+    caseContainerDto.setAddressType(TEST_ADDRESS_TYPE);
+    caseContainerDto.setAddressLevel(ESTAB_ADDRESS_LEVEL);
+    String returnJson = objectMapper.writeValueAsString(caseContainerDto);
+
+    stubFor(
+        get(urlEqualTo(url))
+            .willReturn(
+                aResponse()
+                    .withStatus(HttpStatus.OK.value())
+                    .withHeader("Content-Type", "application/json")
+                    .withBody(returnJson)));
+
+    // When
+    rabbitQueueHelper.sendMessage(caseEventExchange, REFUSAL_ROUTING_KEY, responseManagementEvent);
+
+    // Then
+    assertThat(rabbitQueueHelper.getMessage(outboundQueue)).isNull();
```

Comment:
The rabbit queue helper has a `checkNoMessage` method already which doesn't wait as long (5s vs 20s like this).

Reply:
Nice!

Reply:
@liamtoozer You made the change to the above test but not this one